code-studio/build-js.js

```diff
@@ -31,6 +36,16 @@ build_commands.execute([
     commonFile: 'code-studio-common',
     shouldMinify: commander.min,
     shouldWatch: commander.watch
+  }),
+  build_commands.browserify({
+    srcPath: SRC_PATH + 'initApp/',
+    buildPath: BUILD_PATH,
+    filenames: [
+      'initApp.js'
+    ],
+    commonFile: 'initApp-common.js',
```

Comment:
Do you actually load `initApp-common.js` anywhere?  The modules won't load if you forget to include this file.